### PR TITLE
WIP: Add ability to get source page data

### PR DESCRIPTION
**Pull request by kalebswartz7**
- Allow user to grab data from source page (`intensityReport` which corresponds to spectrum report) instead of beamline

### CODE REVIEW

File: srw_handler.py [hunk 1]
```diff
@@ -2,9 +2,11 @@
 import uti_plot_com as srw_io
 
 
-def read_srw_file(filename):
+def read_srw_file(filename, ndim=2):
```
Comment (mrakitin) [inline srw_handler.py:5]: @danielballan, we need to dynamically pass the number of dimensions to the handler defined below (`SRWFileHandler`), but the problem is that it's registered right after the db is defined in re_config.py, so I'm not sure what's the best way to make this choice dynamic. Any ideas?

Comment (danielballan) [reply srw_handler.py:5]: Can you get away with registering it later?

Comment (mrakitin) [reply srw_handler.py:5]: I think we can. Should we define 2 separate handlers?

Comment (danielballan) [reply srw_handler.py:5]: Seems right. You could use a factory function, but two separate classes are probably easier to document/understand quickly.

**Comment by danielballan** [comment]
:+1: `ndim` code looks good. Nice work.

File: srw_handler.py (hunk 1 above)
Comment (mrakitin) [reply srw_handler.py:5]: OK, thanks @danielballan. I'll review it later.

**Comment by mrakitin** [comment]
@kalebswartz7, thanks for further improvements to the code!

**Comment by mrakitin** [comment]
The 2 commits from this PR went to #9, so closing it.

**Pull request closed without merging**